docs/templates/content/components/inspectors.ftl

```diff
@@ -2,26 +2,15 @@
 
 An inspector is typically a plugin that a ${solution_name} detector uses to access the internal resources of a package manager through its API.
 
-There are currently three inspectors that ${solution_name} might download and one inspector it has internally.
-  
-If ${solution_name} decides that your package manager needs an external inspector, you must either be online or have the applicable air gap files.
-
-## Gradle inspector
```

Comment:
Isn't this all still true but just for integration-common instead of an inspector?

Reply:
it is true you still need to be online or use the air gap Zip but we don't use the gradle inspector library. Wouldn't it just be a Gradle detector now or do we still consider this an inspector?